.github/actions/build/action.yaml

```diff
@@ -111,11 +116,13 @@ runs:
           done
         fi
 
-        # Wait until the MicroShift service is ready and healthy
+        # Start-stop test with readiness check
         make run-ready
         make run-healthy
-
-        # Stop the MicroShift container
+        if [ "${{ inputs.multi-node }}" = "1" ]; then
```

Comment:
Nit: Should this be a count of nodes with default one, i.e. test-clusters-size or test-cluster-nodes?
Then we can write a loop of add-node commands to make it more generic.